microcosm_flask/conventions/encoding.py

```diff
@@ -68,20 +69,22 @@ def load_request_data(request_schema, partial=False):
     try:
         json_data = request.get_json(force=True) or {}
     except Exception:
-        # if `simplpejson` is installed, simplejson.scanner.JSONDecodeError will be raised
+        # if `simplejson` is installed, simplejson.scanner.JSONDecodeError will be raised
         # on malformed JSON, where as built-in `json` returns None
         json_data = {}
-    request_data = request_schema.load(json_data, partial=partial)
-    if request_data.errors:
-        # pass the validation errors back in the context
+    try:
+        return request_schema.load(json_data)
+    except ValidationError as error:
```

Comment:
I main different with 3.0 is that they treat data only as data, errors are explicit exceptions.